draft-ietf-quic-http.md

```diff
@@ -424,13 +424,14 @@ H3_REQUEST_INCOMPLETE.
 A server can send a complete response prior to the client sending an entire
 request if the response does not depend on any portion of the request that has
 not been sent and received. When the server does not need to receive the
-remainder of the request, it MAY abort reading the request stream with error
-code H3_EARLY_RESPONSE, send a complete response, and cleanly close the
-sending part of the stream. Clients MUST NOT discard complete responses as a
-result of having their request terminated abruptly, though clients can always
-discard responses at their discretion for other reasons.  If the server sends a
-partial or complete response but does not abort reading, clients SHOULD continue
-sending the body of the request and close the stream normally.
+remainder of the request, it MAY abort reading the request stream, send a
```

Comment:
I know we try to avoid using QUIC transport level vocabulary in the HTTP/3 document, but I've tripped up on this language before and your change reminded me about it. When it says that the server may "abort reading the stream ... " and use "... H3_NO_ERROR" it *sounds* like it's trying to RST_STREAM the stream, but of course it's really just asking the peer to stop sending (via STOP_SENDING). Would it make sense to phrase this like:

When the server does not need to receive the remainder of the request, it MAY request that the client stop sending on the the request stream, send a complete response, and cleanly close the sending part of the stream.  The error code H3_NO_ERROR SHOULD be used when requesting that the client stop sending on the request stream.

Reply:
The terminology is [in the transport spec](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#rfc.section.2.4), and one of the operations is:
> abort reading of the stream and request closure, possibly resulting in a STOP_SENDING frame (Section 19.5).

I think rephrasing the second sentence to try and improve the linkage would be good, though.

Reply:
Looks great!